ui/src/app/utils/runtime-utils.tsx

```diff
@@ -263,12 +269,8 @@ const compareDataprocNumberOfWorkers = (oldRuntime: RuntimeConfig, newRuntime: R
 };
 
 const compareAutopauseThreshold = (oldRuntime: RuntimeConfig, newRuntime: RuntimeConfig): RuntimeDiff => {
-  const oldAutopauseThreshold = oldRuntime.autopauseThreshold;
-  const newAutopauseThreshold = newRuntime.autopauseThreshold;
-
-  if (!oldAutopauseThreshold || !newAutopauseThreshold) {
-    return null;
-  }
+  const oldAutopauseThreshold = oldRuntime.autopauseThreshold == null ? DEFAULT_AUTOPAUSE_THRESHOLD_MINUTES : oldRuntime.autopauseThreshold;
```

Comment:
Possibly this fixes RW-7244?

Reply:
I saw that too actually but I think it's unrelated to the autopause changes since the bug appears when changing other "update in place" parameters as well. I think there was another PR that in went recently that changed the messaging behavior of the various update types.

I was planning to fix that too but I'll bring it in with a follow up.